Source/SG/SG.cs

```diff
@@ -339,6 +421,7 @@ end repeat
 
             //Download the generation script to the generator for later initiation
             NIRfsgPlayback.SetScriptToGenerateSingleRfsg(rfsgPtr, sb.ToString());
+            rfsgHandle.Arb.SignalBandwidth = waveform.SignalBandwidth_Hz; // has to be manually set since we've disabled it in rfsgpbl

```

Comment:
Why is this explicitly necessary? We do set this on line 286.

Reply:
this was an artifact of pbl resetting this property when i didn't want it to. its removed in the latest code.